src/Tribe/Main.php

```diff
@@ -1610,15 +1610,21 @@ public function addVenueAndOrganizerEditor() {
 		 */
 		public function displayEventVenueDropdown( $post_id ) {
 			$venue_id = get_post_meta( $post_id, '_EventVenueID', true );
+
+			$is_community_edit = false;
+			if ( class_exists( 'Tribe__Events__Community__Main' ) ) {
+				$is_community_edit = Tribe__Events__Community__Main::instance()->isEditPage;
+			}
```

Comment:
Since this is 4 lines with the smallest amount of logic, it feels like it should be a method.